lib/index.js

```diff
@@ -76,6 +77,9 @@ class PWMetrics {
     if (this.opts.json) {
       return data;
     } else if (this.opts.expectations) {
+      if (this.opts.showChart) {
+        this.showChart(data);
+      }
       expectations.checkExpectations(data.timings, this.opts.metrics)
```

Comment:
Just one snit pick. @MickeyKay what do you think if we draw chart after expectations will be checked. Like
```
expectations.checkExpectations(data.timings, this.opts.metrics);
if (this.opts.showChart)  this.showChart(data);
```